### PR TITLE
GameINI: NAC - Legend of Zelda: Ocarina of Time VC - Set EFBToTextureEnable to False

### PR DESCRIPTION
**Pull request by Ecksters**
Fix [issue #9942,](https://bugs.dolphin-emu.org/issues/9942) make Link preview visible in Equipment Menu screen, with EFB to Textures Only enabled the preview simply stays black, rather than showing Link's current equipment.

Before:
EFB Copies Disabled, No Link

After:
EFB Copies Enabled, Link is now visible

### CODE REVIEW

**Comment by JosJuice** [comment]
You have to squash your two commits into one. (This can done using command-line git, but it's not possible in GitHub's web UI.)

I would also like the title (of both the commit and the PR) to be changed so that it includes which game the change is for. That's more important to include in the title than the exact setting that was changed.

File: Data/Sys/GameSettings/NAC.ini [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 [EmuState]
 # The Emulation State. 1 is worst, 5 is best, 0 is not set.
-EmulationStateId = 4
-EmulationIssues = Needs EFB to RAM for the pause menu (background and Link).
+EmulationIssues =
+EmulationStateId = 3
```
Comment (leoetlino) [inline Data/Sys/GameSettings/NAC.ini]: Why was this changed?

Comment (Ecksters) [inline Data/Sys/GameSettings/NAC.ini]: This was changed to follow the settings on the Gamecube version, since apparently they share similar graphical requirements, this was partially why I had two separate commits, because I wasn't sure what the exact effect of that 2nd change was.

**Comment by leoetlino** [comment]
@dolphin-emu-bot rebuild

**Comment by Ecksters** [comment]
Any other changes I need to make?

**Comment by JosJuice** [comment]
I think it looks fine now.

**Comment by Ecksters** [comment]
Aw crap, I think I may have over complicated things by trying to update my branch with the changes from Master

**Comment by leoetlino** [comment]
Yes, this isn't properly rebased now… Since it's a two lines diff, the easiest way to fix this would probably be to `git reset --hard master` and redo the changes manually.

**Comment by Ecksters** [comment]
https://github.com/dolphin-emu/dolphin/pull/4508

Ugh, probably a better way that making a new PR, but just seemed easier to do this.
